src/fuel_fab.h

```diff
@@ -199,9 +199,10 @@ class FuelFab : public cyclus::Facility {
   #pragma cyclus var { \
     "doc": "Top-up material stream request preference.", \
     "uilabel": "Top-up Stream Preference", \
-    "default": 0, \
+    "default": 0.01, \
```

Comment:
done

On Tue, Aug 25, 2015 at 2:52 PM, Robert Carlsen <notifications@github.com>
wrote:

> In src/fuel_fab.h
> <https://github.com/cyclus/cycamore/pull/381#discussion_r37910995>:
>
> > @@ -199,9 +199,10 @@ class FuelFab : public cyclus::Facility {
> >    #pragma cyclus var { \
> >      "doc": "Top-up material stream request preference.", \
> >      "uilabel": "Top-up Stream Preference", \
> > -    "default": 0, \
> > +    "default": 0.01, \
>
> s/eyesore/sore thumb/
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cyclus/cycamore/pull/381/files#r37910995>.
>



-- 
Matthew Gidden, Ph.D.
Postdoctoral Associate, Nuclear Engineering
The University of Wisconsin -- Madison
Ph. 225.892.3192